core/sql/src/main/java/org/trafodion/sql/HBaseClient.java

```diff
@@ -294,15 +296,27 @@ private ChangeFlags setDescriptors(Object[] tableOptions,
                break ;
            case HBASE_COMPRESSION:
                if (tableOption.equalsIgnoreCase("GZ"))
+	       {    // throws IOException
+		   CompressionTest.testCompression(Algorithm.GZ);
                    colDesc.setCompressionType(Algorithm.GZ);
+	       }
                else if (tableOption.equalsIgnoreCase("LZ4"))
+	       {   // throws IOException
+		   CompressionTest.testCompression(Algorithm.LZ4);
                    colDesc.setCompressionType(Algorithm.LZ4);
+	       }
                else if (tableOption.equalsIgnoreCase("LZO"))
+	       {   // throws IOException
+		   CompressionTest.testCompression(Algorithm.LZO);
                    colDesc.setCompressionType(Algorithm.LZO);
+	       }
                else if (tableOption.equalsIgnoreCase("NONE"))
                    colDesc.setCompressionType(Algorithm.NONE);
                else if (tableOption.equalsIgnoreCase("SNAPPY"))
+	       {   // throws IOException
+		   CompressionTest.testCompression(Algorithm.SNAPPY);
                    colDesc.setCompressionType(Algorithm.SNAPPY); 
+	       }
```

Comment:
Should there be an exception if the tableOption isn't recognized? (Right now the code just ignores it.)